Makefile

```diff
@@ -72,6 +72,10 @@ bdd-test-js-local:
 vc-test-suite: clean
 	@scripts/run_vc_test_suite.sh
 
+.PHONY: bss-interop-test
+bss-interop-test:
```

Comment:
can you create CI build for this

Reply:
we need to make sure it's passing in CI

Reply:
like test-suite / VC test suite

Reply:
Added bbs-interop / BBS+ interop test.
Thanks @fqutishat for the proposal.